src/commandsAndMenu.tsx

```diff
@@ -60,6 +60,11 @@ export interface IGitCloneArgs {
    * Git repository url
    */
   url: string;
+  /**
+   * Whether to activate git not_versioning in the clone or not.
+   * If false, this will remove the .git folder after cloning.
+   */
+  not_versioning: boolean;
```

Comment:
```suggestion
   * Whether to activate git versioning in the clone or not.
   * If false, this will remove the .git folder after cloning.
   */
  versioning: boolean;
```